src/components/updatemenus/draw.js

```diff
@@ -54,7 +54,7 @@ module.exports = function draw(gd) {
      */
 
     // draw update menu container
-    var menus = fullLayout._infolayer
+    var menus = fullLayout._menulayer
```

Comment:
Ha I guess that does it. I first thought about splitting the updatemenu buttons from their header `<g>` i.e. the header would have remained in `<g infolayer>` and the buttons moved to `<g menulayer>`.

Can anyone think of a situation where one would want to place an updatemenu below other `<g infolayer>` features?

Reply:
I can't see when you'd want the updatemenu below anything else, but I guess it could be useful to separate `<g menulayer>` itself into two pieces, so that regardless of the order you draw the menus, when they open up the transient parts will be on top. The only case I can think of that would really *need* this is if you have two menus that each overlaps the other when opened - one above that opens downward and one below that opens upward for example. If you have a lot of menus that might become necessary at times.

Reply:
Updatemenus were already smart enough to put the dropdown buttons (but not always-visible buttons) on top - exactly what I was suggesting 🎉 🏆 But there was a bug with this that if a new menu was added when other menus already existed, this would get added on top of the dropdown buttons -> fixed in https://github.com/plotly/plotly.js/pull/2121/commits/e0a10af575aa864365248a1e210961fb02bf8092